test/unit/custom_spec.js

```diff
@@ -107,3 +107,119 @@ describe("custom canvas rendering", function () {
       .catch(done.fail);
   });
 });
+
+describe("alternate document context", function () {
+  const FontFace = global.FontFace;
```

Comment:
This is completely wrong, since `global` isn't actually defined other than in Node.js and these unit-tests now fail in browsers with `ReferenceError: global is not defined ...`

Fixing that locally, however, *still* causes these tests to fail, now with `TypeError: canvas.getContext is not a function ...` in browsers.

When reviewing this, I simply assumed that the new unit-tests were actually *tested* and thus working, but looking at them again I'm no longer sure that they're doing the right thing in general unfortunately. (E.g. the `ownerDocument` probably *needs* to be a valid `HTMLDocument`-instance, rather than a regular object.)

It's strange that this didn't cause the bots to fail, but it's clear from the logs that these new tests didn't even run. Unfortunately I cannot see an *immediate* fix for the unit-tests, and just removing the unit-tests seem like the wrong approach.

@timvandermeij Can you please back out this patch, since this shouldn't have landed as-is.

Reply:
This patch is now reverted. It's indeed really not great that Jasmine doesn't even run the test and therefore the bots think all is fine. I'll try to find out why that happens since we want to get a failed test if that ever happens.